dist/s_all

```diff
@@ -85,16 +125,21 @@ errchk()
 
 run()
 {
+    local bg=
+    [[ "$1" == "--bg" ]] && { bg=1; shift; }
     local cmd="$1" flags="$2"
     local name="${cmd%% *}"
     local t="${t_pfx}-${name}-$$"
     [[ "$name" =~ .*\.py$ ]] && cmd="python3 $cmd" || cmd="/bin/bash $cmd"
+    [[ -z "$bg" ]] && status "Running: $name"
     $cmd > $t 2>&1
+    [[ -z "$bg" ]] && status ""
     errchk "$name" "$t" "$flags"
 }
 
-# Non parallelizable scripts. The following scripts either modify files or
-# already parallelize internally.
+# Non parallelizable scripts.
+# The following scripts either modify files or already parallelize internally.
+run "s_version $force"     # Update files that include the package version.
```

Comment:
I wonder why this was executed separately before.

Reply:
Good point. [Here](s_versionhttps://github.com/wiredtiger/wiredtiger/commit/4c25ead104a289e3b2275b7e84b4a37ed1512cb6)'s an answer. Moving it out of `run` thing.

Reply:
.. although I'd rather make it behave like all other scripts